# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../kobo_deluxe/gcc4.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur KoboDeluxe-0.4pre10-orig/gamestate.h KoboDeluxe-0.4pre10/gamestate.h
--- KoboDeluxe-0.4pre10-orig/gamestate.h	2002-04-28 01:48:59.000000000 +0000
+++ KoboDeluxe-0.4pre10/gamestate.h	2006-01-25 20:12:28.745277750 +0000
@@ -24,6 +24,8 @@
 
 #include "window.h"
 
+class gamestatemanager_t;
+
 class gamestate_t
 {
 	friend class gamestatemanager_t;
diff -Nur KoboDeluxe-0.4pre10-orig/scenes.h KoboDeluxe-0.4pre10/scenes.h
--- KoboDeluxe-0.4pre10-orig/scenes.h	2001-09-02 23:09:18.000000000 +0000
+++ KoboDeluxe-0.4pre10/scenes.h	2006-01-25 20:07:55.316189500 +0000
@@ -39,7 +39,7 @@
     int x, y, h, v;
 } _base;
 
-typedef const struct {
+typedef const struct _scene_struct {
     int ratio;
     int startx;
     int starty;
